source/beamline/AMControl.cpp

```diff
@@ -54,3 +54,11 @@ bool AMControl::hasChildControl(AMControl *control) const
 
 	return result;
 }
+
+void AMControl::removeChildControl(AMControl *control)
```

Comment:
I agree, and I would like addChild() and removeChild() to return bools. Not sure how best to go about it, as they are used and reimplemented quite a lot in AMPseudoMotorControl subclasses. Should that be done in this pull request, or as something separate?